docs/fedora.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Using mkosi-initrd with Fedora
+SPDX-License-Identifier: LGPL-2.1-or-later
+---
+
+# Building an initramfs image
+
+Two tools are required: `mkosi`, currently a git version is required,
+and the `mkosi-initrd` repository with configuration for `mkosi`.
+
+```bash
+git clone https://github.com/keszybz/mkosi
```

Comment:
I think there should be git clone https://github.com/systemd/mkosi your clone does not have RemovePackages for example